src/System.Net.WebProxy/tests/WebProxyTest.cs

```diff
@@ -145,7 +145,7 @@ public static IEnumerable<object[]> BypassOnLocal_MemberData()
             yield return new object[] { new Uri($"http://{Guid.NewGuid().ToString("N")}"), true };
             foreach (IPAddress address in Dns.GetHostEntryAsync(Dns.GetHostName()).GetAwaiter().GetResult().AddressList)
             {
-                if (address.AddressFamily == AddressFamily.InterNetwork)
+                if (address.AddressFamily == AddressFamily.InterNetwork && !IPAddress.IsLoopback(address))
```

Comment:
@cipop, is the right fix to address this in the test like this, or should this be done inside of GetHostEntryAsync?